### PR TITLE
Chinese files

### PR DESCRIPTION
**Pull request by GayathriVenkatesh**
Added files for Chinese translation as per issue #1345

### CODE REVIEW

**Comment by GayathriVenkatesh** [comment]
Hi @julianguyen,
@Manasa2850 and I worked on issue #1345 and fixed the circle ci error. Do let us know if this is ok :D

**Comment by GayathriVenkatesh** [comment]
Not yet, I don't know anyone who speaks Chinese.
It would be great if someone from the if-me community helps us out!

File: config/locales/zh.yml [hunk 1]
```diff
@@ -0,0 +1,732 @@
...
+    pt-BR: Português
+    sv: Svenska
+    various_languages: 'Various languages'
+    vi: 'Tiếng Việt'
+    it: Italiano
+    nb: Norsk
+    de: Deutsch
+    fr: Français
+    hi: 'हिंदी'
+    zh: 中文
```
Comment (andy5995) [inline config/locales/zh.yml]: Does this line need to be added to the other locale files?

Comment (andy5995) [reply config/locales/zh.yml]: Oh, nevermind. I just spot-checked 3 of them and see it's there.

File: config/locales/zh.yml [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+      premade4_name: 不满
+      premade4_description: '不接受某些事实，生气，报仇和/或嫉妒'
+      premade5_name: 绝望
+      premade5_description: '感到悲伤和绝望-无法找到任何可能的解决方案'
+      instructions: '您尚未创建任何自定义心情。<br>您还可以添加以下预制的情绪，然后再次自定义它们。'
+  notifications:
+    plural: 通知事项
+    none: '没有了'
+    clear: 明确
+    created_by_html: '<strong>Created by</strong>: %{name}'
```
Comment (andy5995) [inline config/locales/zh.yml]: Does "Created by" need translation?

Comment (GayathriVenkatesh) [reply config/locales/zh.yml]: Sorry, my bad! Will fix it now :)

**Comment by julianguyen** [comment]
Since it's taking a while to get feedback, I'm just going to merge this. Folks can always commit pull requests with translations improvements/fixes! Thanks so much for this fantastic work!

**Pull request merged**